packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -263,6 +251,34 @@ function parseStringCommandOption(optionValue: any): string | undefined {
   return (typeof(optionValue) === "string") ? optionValue : undefined;
 }
 
+export function parseWebView(webViewText?: string): devSettings.WebViewType | undefined {
+  const parsedWebView = webViewText ? webViewText.toLowerCase() : undefined;
+  switch (parsedWebView) {
```

Comment:
you don't really need the variable name here. I think it would be better to move the conditional expression directly into the switch.

`switch (webViewType ? webViewText.toLowercase() : undefined)`